axelor-admin/src/main/java/com/axelor/apps/base/service/app/DataBackupCreateService.java

```diff
@@ -491,6 +483,8 @@ protected CSVInput writeCSVData(
         csvInput.setSearch("self.importId = :importId");
       }
     } catch (ClassNotFoundException e) {
+    } catch (AxelorException e) {
+      throw new AxelorException(TraceBackRepository.CATEGORY_CONFIGURATION_ERROR, e.getMessage());
```

Comment:
If we already have an AxelorException, we do not need to recreate it.

If we have a classnotfound exception, we probably need to use a constructor that use a throwable to not lose the stacktrace: `AxelorException(e, TraceBackRepository.CATEGORY_CONFIGURATION_ERROR, e.getMessage())`